lua/entities/ent_jack_turret_base/init.lua

```diff
@@ -102,7 +112,16 @@ function ENT:ExternalCharge( amt )
 end
 
 function ENT:Initialize()
-    self:SetAngles( Angle( 0, 0, 0 ) )
+
+    local spawnAng = Angle( 0, 0, 0 )

```

Comment:
Can you move this var outside of this function?

Reply:
Applied!